addons/mass_mailing/models/mailing.py

```diff
@@ -658,3 +671,68 @@ def _image_to_url(b64image: bytes):
             return lxml.html.tostring(root)
 
         return body_html
+
+    def _send_statistics_mails(self):
+        """Send an email to the responsible of each finished mailing with the statistics."""
+        mailings = self.env['mailing.mailing'].search([
+            ('kpi_mail_required', '=', True),
+            ('state', '=', 'done'),
+            ('sent_date', '<=', fields.Datetime.now() - relativedelta(minutes=5)),
+            ('sent_date', '>=', fields.Datetime.now() - relativedelta(days=3)),
+        ])
+
+        if not mailings:
+            return
+
+        mail_template = self.env.ref('mass_mailing.mail_template_mass_mailing_kpi')
+        if not mail_template:
+            _logger.error('Mass Mailing: can not found the mass mailing KPI template')
+            return
+
+        mailings.kpi_mail_required = False
+
+        for mailing in mailings:
+            email_values = {'email_to': mailing.user_id.email}
+            mail_template.send_mail(mailing.id, email_values=email_values, force_send=True)
+
+    def prepare_statistics_email_values(self):
```

Comment:
As template will be moved to qweb, I think we can already put this method private and put its data in context. Then it will be used directly in qweb rendering.